rules/bunit-for-blazor-unit-tests/rule.md

```diff
@@ -61,15 +64,15 @@ public class ExampleComponentTests
 }
 ```
 ::: good
-Figure: Good example - Using BUnit to test a Blazor component
+Figure: Good example - Using bUnit to test a Blazor component
 
 :::
 <br>
 
-This is a very simple example, but the same concepts apply to more complex components. BUnit also provides a number of other features that make it easier to write unit tests for Blazor components, such as the ability to mock services and inject them into components.
+This is a very simple example, but the same concepts apply to more complex components. bUnit also provides a number of other features that make it easier to write unit tests for Blazor components, such as the ability to mock services and inject them into components.
 
-Complex components such as complicated searching and filtering are good candidates for BUnit tests, to ensure that a component behaves as expected.
+Complex components such as complicated searching and filtering are good candidates for bUnit tests, to ensure that a component behaves as expected.
 
 ### References
 
-- [Microsoft Learn: Unit Testing Blazor Components with BUnit](https://learn.microsoft.com/en-us/aspnet/core/blazor/test?view=aspnetcore-7.0)
+- [Microsoft Learn: Unit Testing Blazor Components with bUnit](https://learn.microsoft.com/en-us/aspnet/core/blazor/test?view=aspnetcore-8.0)
```

Comment:
just don't link to any view @bradystroud @MattParkerDev 

> https://learn.microsoft.com/en-us/aspnet/core/blazor/test

Reply:
done :)